srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/Pages/SoapLogs.php

```diff
@@ -0,0 +1,146 @@
+<?php
+    use ACore\Manager\Opts;
+    // Now display the settings editing screen
+    $myCredConfs = get_option('mycred_pref_core');
+?>
+
+<div class="wrap">
+    <h2> <?= __('Soap Logs', Opts::I()->page_alias)?> </h2>
+    <p>A list of the logged commands.</p>
+    <div class="row">
+        <div class="col">
+            <div class="card p-0">
+                <div class="card-body">
+                    <h5>Soap Logs</h5>
+                    <hr>
+                    <form class="row row-cols-lg-auto g-2" url="<?php echo menu_page_url(ACORE_SLUG . '-soap-logs'); ?>">
+                        <input type="hidden" value="<?php echo ACORE_SLUG . '-soap-logs'; ?>" name="page">
+                        <div class="col">
+                            <label class="visually-hidden" for="username">Username</label>
+                            <input type="text" class="form-control form-control-sm" id="username" name="username" placeholder="Username" value="<?php echo $data['username']; ?>">
+                        </div>
+                        <div class="col">
+                            <label class="visually-hidden" for="order_id">Order ID</label>
+                            <input type="text" class="form-control form-control-sm" id="order_id" name="order_id" placeholder="Order ID" value="<?php echo $data['order_id']; ?>">
+                        </div>
+
+                        <div class="col">
+                            <label class="visually-hidden" for="items">Items</label>
+                            <select class="form-select" name="items" id="items">
+                                <option value="10">10</option>
+                                <option value="25" <?php echo $data['items'] == '25' ? 'selected' : ''; ?>>25</option>
+                                <option value="50" <?php echo $data['items'] == '50' ? 'selected' : ''; ?>>50</option>
+                                <option value="100" <?php echo $data['items'] == '100' ? 'selected' : ''; ?>>100</option>
+                            </select>
+                        </div>
+
+                        <div class="col">
+                            <button type="submit" class="btn btn-primary btn-sm">Search</button>
+                        </div>
+
+                        <div class="col ms-auto">
+                            <nav aria-label="Table pagination">
+                                <ul class="pagination pagination-sm">
+                                    <?php
+                                        $maxPagination = 3;
+                                        $diff = $data["max_page"] - $data["pos"];
+                                        $start = $data["pos"] - $maxPagination > 0 && $diff >= 0 ? $data["pos"] - $maxPagination : 1;
```

Comment:
Would be better to add some `()` when the condition in the ternary operator is more than one